apps/labs/components/Projects/ProjectHeadline.tsx

```diff
@@ -7,16 +7,16 @@ export const ProjectHeadline = ({
   toggleItemDropdown,
   title,
 }: TProjectHeadlineProps) => (
-  <div className="flex justify-between items-center w-full sm:col-start-1 sm:col-end-2">
-    <h3 className="text-[4rem] font-extrabold leading-[6.648rem] font-heading text-violet">
+  <div className="flex relative justify-between items-center w-full sm:col-start-1 sm:col-end-2">
+    <h2 className="shrink-0 text-[4rem] font-extrabold leading-[6.648rem] font-heading text-violet">
       {title}
-    </h3>
+    </h2>
     <button
       onClick={toggleItemDropdown}
-      className="flex relative justify-center items-center w-[3rem] h-[3rem]"
+      className="flex absolute flex-row-reverse items-center w-[100%] h-[6rem]"
```

Comment:
I thought the button hit area was too small, so I expanded it to take up the whole row.